circuit-types/src/lib.rs

```diff
@@ -37,13 +38,25 @@ use wallet::{Wallet, WalletShare};
 // | Constants |
 // -------------
 
+/// The zero value in the Scalar field we work over
+pub const SCALAR_ZERO: ScalarField = ScalarField::new(BigInt::new([0, 0, 0, 0]));
+/// The one value in the Scalar field we work over
+pub const SCALAR_ONE: ScalarField = ScalarField::new(BigInt::new([1, 0, 0, 0]));
```

Comment:
Confirming this is correct even in Montgomery form?

Reply:
The `BigInt` constructor is not concerned with montgomery form, only the `Fp` (which is a `MontgomeryBackend`) will make such changes to the underlying bigint